src/components/DIDSessionDemoHandler.tsx

```diff
@@ -19,24 +20,44 @@ const MessageTextField = styled(TextField)({
 
 function DIDSessionDemoHandler() {
   const [session, setSession] = useState<DIDSession>()
+  const resources = ['test-resource', 'another-test-resource']
 
-  const authenticate = async () => {
+
+  const signInWithEthereum = async () => {
     const ethProvider = await detectEthereumProvider();
-    const addresses = await (window.ethereum as any).request({ method: 'eth_requestAccounts' })
-    const authProvider = new EthereumAuthProvider(ethProvider, addresses[0])
-
-    const oneWeek = 60 * 60 * 24 * 7
-
-    const session = await DIDSession.authorize(
-      authProvider,
-      {
-        resources: ['test-resource', 'another-test-resource'],
-        expiresInSecs: oneWeek,
-        domain: 'YourAppName'
-      })
+    const addresses = await (ethProvider as any).enable()
```

Comment:
I think this is the proper way to do it still!
```suggestion
    const addresses = await (ethProvider as any).request({ method: 'eth_requestAccounts' })
```

Reply:
We should probably update the docs.

Reply:
Will do! Thanks, Joel!